### DISCUSSION TITLE
Receiving release email notifications via GitHub

### DISCUSSION THREAD

**Discussion by dantefromhell** [nextcloud/all-in-one#1983] [Questions] [answered]
I'm observing that although I have set my watch settings for this repo to 'Releases' I only receive `Pre-Release` notifications but not `Release` notifications as I do for other github projects.

I've been observing/ tracking this behaviour for the past few months, e.g. I received notifications for 4.3.4 Beta/ 4.3.3 Beta/ 4.3.2 Beta/ 4.3.1 Beta/ 4.3.0 Beta/ 4.2.0 Beta/ 4.1.0 Beta but not for the actual `4.2.0` release.

Is anyone else observing this behaviour?

**Comment by szaimen** [comment, marked as answer]
Hi, I think the problem is that Github does not send notifications when changing a pre-release to a stable release so something Github needs to fix or to implement. I had the same problem e.g. with https://github.com/mozilla/pdf.js

**Reply by dantefromhell** [reply]
I'm fairly certain this is not a github issues - I get the expected notifications for the `hashicorp/terraform` repo.

Comparing their release page [1] and the `nextcloud-aio` release page [2] the first difference that stands out for me, is that aio has releases whose name ends with "BETA" and the `Pre-Release` marker, but there are no `stable` (or not `Pre-Release`) marked releases for the same version number.

Is this could be part of the `stable` vs `beta` version mechanic? I was not able to find documentation pointing in this direction?
If yes, is it intended that certain semantic versions wil never be promoted into the `stable` channel?

[1] https://github.com/hashicorp/terraform/releases
[2] https://github.com/nextcloud/all-in-one/releases
